src/client/core/utils/dom.js

```diff
@@ -443,3 +443,9 @@ export function findIframeByWindow (iframeWindow, iframeDestinationWindow) {
 
     return null;
 }
+
+export function canAcceptValue (element) {
```

Comment:
are you sure about this `accept value` ?
what do these elements have in common? is this about 'accepting values'?